script.js

```diff
@@ -27,3 +28,19 @@ for (let i = 0; i < links.length; i += 1) {
     navbarMenu.classList.add('display-none');
   });
 }
+
+const form = document.querySelector('.contact-form');
+const errorMsg = document.getElementById('text');
+const email = document.getElementById('email');
+
+form.addEventListener('submit', (event) => {
+  const pattern = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+  if (email.value.match(pattern)) {
+    errorMsg.innerHTML = 'This is a valid Email';
+    errorMsg.style.color = '#00ff00';
+  } else {
+    event.preventDefault();
+    errorMsg.innerHTML = 'Not a valid Email';
+    errorMsg.style.color = '#ff0000';
+  }
+});
```

Comment:
- [ ] [OPTIONAL] Please, if you can add more styles to the error notification, it would be nice to add due to right now it doesn't look properly.
- [ ] [OPTIONAL] It would be nice if you add a more descriptive error: "Error Email Not Valid: You must use lowercase in the email field"

![image](https://user-images.githubusercontent.com/60164239/136285435-9364bcb7-650c-49f4-8d29-a75adb748c07.png)